graphstorm-processing/graphstorm_processing/constants.py

```diff
@@ -19,9 +19,6 @@
 MISSING_CATEGORY = "GSP_CONSTANT_UNKNOWN"
 SINGLE_CATEGORY_COL = "SINGLE_CATEGORY"
 
-################ Multi-numerical Limits #####################
-MAX_COLUMNS_TO_IMPUTE = 50
```

Comment:
We had an alternative implementation for multi-numerical that used a different codepath when we had multiple columns. I removed it here for simplicity, if scaling becomes an issue we can bring it back (it wasn't much help anyway). 

The original cause for this is that to apply imputations to a vector df (where each entry is a numerical vector) we needed to first convert each value in the vector to a column, then apply the imputation to each column. This would often trigger OOM errors when the dimensions are large.